src/css/components/_c-person.scss

```diff
@@ -18,53 +19,60 @@ $_person-photo: (
 	}
 }
 
+@mixin photo($size: tiny) {
+	height: map-get($_person-photo, $size);
+	width: map-get($_person-photo, $size);
+}
+
 
 // Photos use background images since we can't control the dimensions of
 // submitted avatars. This centers whatever is added, which hopefully creates a
 // good cropping effect.
 .c-person__photo {
+	@include photo(tiny);
+
 	background-position: center;
 	background-size: contain;
-	border-radius: map-get($_person-photo, small) * 2;
+	border-radius: 100%;
 	cursor: pointer;
 	display: inline-block;
-	height: map-get($_person-photo, small);
-	margin-top: -3.5rem;
+	margin-top: -2.75rem;
 	position: absolute;
-		right: 0.25rem;
-	width: map-get($_person-photo, small);
+		right: calc(50% - #{map-get($_person-photo, tiny)/2});
 
 	// Breakpoints
 	@include mappy-bp(wrist-large) {
-		right: 6vw;
+		@include photo(small);
```

Comment:
Another style thing we're doing is using a blank line between mixin includes and declaration blocks. This all gets compiled down to minified CSS, so we don't mind the extra space here in the Sass.

Reply:
I made an adjustment on this front, but let me know if I've put the space on the wrong side of things (if possible, could you use the "suggestion" feature to show me what you mean if I've misinterpreted it? Thank you!)